spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -452,6 +452,7 @@ object SpdxDeclaredLicenseMapping {
         "The Eclipse Public License Version 1.0" to EPL_1_0.toExpression(),
         "The GNU General Public License (GPL), Version 2, With Classpath Exception"
                 to (GPL_2_0_ONLY with CLASSPATH_EXCEPTION_2_0),
+        "The GNU General Public License, v2 with FOSS exception" to GPL_2_0_OR_LATER.with(UNIVERSAL_FOSS_EXCEPTION_1_0),
```

Comment:
For consistency, use the same syntax as above, with `with` as infix function.

Reply:
Done.